arch/platform/nrf52840/dk/Makefile.dk

```diff
@@ -0,0 +1,2 @@
+CONTIKI_TARGET_DIRS += dk dk/dev
```

Comment:
To be honest given how small the number of files here, I'd have dropped everything under `.` without the `dev` subdir